styles.css

```diff
@@ -1 +1,68 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container {
+    background-color: hsl(62, 94%, 89%);
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows:50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
```

Comment:
Ik dacht dat we hier de verschillende blocken moesten beschrijven wat allemaal "naast elkaar" komt staan. Helaas heb ik niet zo veel verstand van... Alleen kan ik zien dat ik als volgende heb gedaan:    grid-template-areas:
    "A C C"
    "B C C"
    "B E D"
    "B E ."
    "F F F";
Kan je mij uitleggen hoe jouw grid-template-areas werken?